apps/weshipit/pages/index.tsx

```diff
@@ -45,6 +50,10 @@ export default function IndexPage({ clients }: clientsPageProps) {
           </Button>
         </div>
       }
+      seoTitle={'React Native Experts'}
```

Comment:
instead of duplicating the content you can use `metadata.title` and `metadata.description`.